tests/string.cpp

```diff
@@ -4,7 +4,7 @@
 
 using namespace Hyprutils::String;
 
-int main(int argc, char** argv, char** envp) {
```

Comment:
why remove my signature trademark `int argc char** argv char** envp`

Reply:
they arent being used

Reply:
but it's a trademark of mine

Reply:
???

Reply:
[a922578](https://github.com/hyprwm/hyprutils/pull/25/commits/a922578e9c6acca4f932d20fc39dd5d574980291) added back here